### ISSUE TITLE
Unneeded whitespaces displayed in response

### ISSUE THREAD

**Issue by muttleyxd** [mrin9/RapiDoc#58]
https://mrin9.github.io/RapiDoc/examples/example2.html

just pick any endpoint and press TRY

![Screenshot from 2019-10-04 09-43-38](https://user-images.githubusercontent.com/9843181/66190086-82c06400-e68b-11e9-9c95-212f301f4ee0.png)

If response is multiline text then it displays like that:
```
            start of my text
another line
last line
< whitespaces here>
```

**Comment by mrin9** [comment]
got that, thanks for reporting
will get that fixed soon

**Comment by mrin9** [comment]
fixed in release `5.3.1`
